src/components/Departments.jsx

```diff
@@ -1,7 +1,54 @@
-import React from "react";
+import React, { useEffect } from "react";
+import { DataGrid } from "@mui/x-data-grid";
+import { useSelector } from "react-redux";
+import { Department } from "../modules/apiHelpers/departmentHelper";
+import Container from "@mui/material/Container";
+import { Link } from "react-router-dom";
 
 const Departments = () => {
-  return <h1>Departments</h1>;
+  const { departments } = useSelector((state) => state);
+
+  useEffect(() => {
+    Department.index();
+  }, []);
+
+  let rows = departments.map((department) => {
+    return {
+      id: department.id,
```

Comment:
Do we need to have this line of code if the columns only include name and score?

Reply:
Disregard, keep DataGrid happy